ArduCopter/mode_auto.cpp

```diff
@@ -343,63 +342,70 @@ void ModeAuto::land_start(const Vector2f& destination)
     copter.ap.prec_land_active = false;
 }
 
-// auto_circle_movetoedge_start - initialise waypoint controller to move to edge of a circle with it's center at the specified location
-//  we assume the caller has performed all required GPS_ok checks
-void ModeAuto::circle_movetoedge_start(const Location &circle_center, float radius_m)
+Mode::_Circling Mode::circling;
```

Comment:
This would go in `circling.cpp` after we merge this PR. A lot of code is already here in wrong namespace to make reviewing easy. This can be fixed when we move this code to right place. If you still need it to be done now, I'll be happy to do it.